features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -205,6 +205,8 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
     const [ isDisableInProgress, setIsDisableInProgress ] = useState<boolean>(false);
     const [ enableStatus, setEnableStatus ] = useState<boolean>(false);
     const [ showDisableConfirmationModal, setShowDisableConfirmationModal ] = useState<boolean>(false);
+    const brandingDisabledFeatures: string[] = window[ "AppUtils" ]?.getConfig().ui?.features?.branding?.
```

Comment:
Let's get this from the redux store. Avoid using `window` object whenever possible

Reply:
Fixed https://github.com/wso2/identity-apps/pull/6409/commits/1f0ef2552c269b4a70a36b8a12bb542f60fc1e78